test/integration/dialog-menu-bar.test.js

```diff
@@ -1,8 +1,9 @@
 import { expect } from '@vaadin/chai-plugins';
 import { fixtureSync, nextRender, oneEvent } from '@vaadin/testing-helpers';
 import '@vaadin/dialog';
-import '@vaadin/menu-bar';
-import '@vaadin/vertical-layout';
+import '@vaadin/menu-bar/test/menu-bar-test-styles.js';
+import '@vaadin/menu-bar/src/vaadin-menu-bar.js';
```

Comment:
I left the `@vaadin/dialog` import since the test explicitly requires Lumo opening animation.